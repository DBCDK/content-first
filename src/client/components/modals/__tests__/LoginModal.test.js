/* eslint-disable no-undefined */
import React from 'react';
import {LoginModal} from '../LoginModal.component';
import renderer from 'react-test-renderer';

jest.mock('../Modal.component', () => 'Modal');

describe('LoginModal', () => {
  it('renders function', () => {
    const tree = renderer
      .create(<LoginModal context={{title: 'abc', reason: 'def'}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
