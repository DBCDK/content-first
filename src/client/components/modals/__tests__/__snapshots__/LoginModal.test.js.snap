// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginModal renders function 1`] = `
<Modal
  className="short-list--merge-modal"
  doneText="LOG IND"
  header="abc"
  onClose={undefined}
  onDone={undefined}
>
  <p>
    <strong>
      def
    </strong>
  </p>
  <small>
    <p>
      Har du ikke en profil? Du kan nemt oprette en profil med det login, du bruger på biblioteket.
    </p>
    <p>
      Klik på login for at oprette en profil.
    </p>
  </small>
</Modal>
`;
